setup.py

```diff
@@ -23,7 +23,9 @@
       install_requires=[
           "PyTango>=9.2.2",
           "numpy",
-          "jsonschema"],
+          "jsonschema",
+          "future",
+          "futures; python_version<'3'"],
```

Comment:
This is not required for stage 1.  We are not using `future` or `futures` in this PR, and `from __future__ ` is standard.

Reply:
If that is the case then it should also be removed from that Futurize stage 1 step in the migration guide, and placed in the relevant step

Reply:
Sorry - I didn't notice that error in the migration guide.